wazo_call_logd/database/models.py

```diff
@@ -108,6 +118,44 @@ class CallLog(Base):
     )
 
 
+@generic_repr
+class RemoteParty(Base):
```

Comment:
Do we really need another table? Could we simply add two fields in the `call_log` table?